src/measure_theory/measure/measure_space_def.lean

```diff
@@ -376,6 +376,14 @@ lemma ae_eq_set {s t : set α} :
   s =ᵐ[μ] t ↔ μ (s \ t) = 0 ∧ μ (t \ s) = 0 :=
 by simp [eventually_le_antisymm_iff, ae_le_set]
 
+lemma ae_eq_set_iff_symm_diff {s t : set α} :
+  s =ᵐ[μ] t ↔ μ (s ∆ t) = 0 :=
+by simp [ae_eq_set, symm_diff_eq_sdiff_sup_sdiff]
```

Comment:
What about turning it around and simping?

Reply:
Good idea, thanks.